src/modules/10-common/components/ShellScriptMonaco/ShellScriptMonaco.tsx

```diff
@@ -143,4 +143,4 @@ export function ShellScriptMonaco(props: ConnectedShellScriptMonacoProps): React
   )
 }
 
-export const ShellScriptMonacoField = connect<ShellScriptMonacoProps>(ShellScriptMonaco)
+export const ShellScriptMonacoField = connect<ShellScriptMonacoProps>(ShellScriptMonaco as any)
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
export const ShellScriptMonacoField = connect<ShellScriptMonacoProps>(ShellScriptMonaco as unknown)
```